src/locales/fi.json

```diff
@@ -697,7 +727,25 @@
       },
       "berths": {
         "title": "Venepaikat",
-        "noBerths": "Sinulla ei ole venepaikkoja"
+        "noBerths": "Sinulla ei ole venepaikkoja",
+        "berthOffer": {
+          "acceptAndPay": "Hyväksy ja siirry maksamaan",
+          "appliedBerths": "Haetut venepaikat",
+          "basePrice": "Perushinta ({{berthWidth}})",
+          "berthNumber": "Paikka",
+          "berthTotal": "Venepaikka yhteensä",
+          "deleteApplication": "Poista hakemus",
+          "dockingEquipmentSummerStorage":"Telakointi\u00ADtarvikkeiden kesäsäilytys",
+          "editApplication": "Muokkaa hakemusta",
+          "harborMap": "Satamakartta (PDF)",
+          "harborWebsite": "Toimipisteen nettisivut",
+          "heading": "Tarjous venepaikasta. Jos haluat vastaanottaa sen,\n maksa se ennen eräpäivää {{date}}.",
+          "infoText": "Hakemus poistetaan molemmissa tapauksissa (hyväksy/hylkää). Mikäli haluat eri paikan, hyväksy tarjous jonka jälkeen voi tehdä vaihtohakemuksen. Vaihtohakemukset käsitellään vuosittain ennen uusia hakemuksia.",
+          "netPrice": "NETTOHINTA",
+          "keepApplicationActive": "Haluan että hakemus jää voimaan mikäli paikkoja ei ole vapaana tälle purjehduskaudelle",
+          "rejectOffer": "Hylkää tarjous",
+          "sportsServicesPrices": "Liikuntapalvelujen hinnasto"
```

Comment:
For the sake of consistency, these keys could be replaced with the snake-case format.